gradle/libs.versions.toml

```diff
@@ -0,0 +1,41 @@
+[versions]
+agp = "7.1.2"
+bytebuddy = "1.12.10"
+jcodec = "0.2.5"
+kotlin = "1.6.10"
+moshi = "1.13.0"
+
+[libraries]
+# Gradle plugins
+androidGradlePlugin = { module = "com.android.tools.build:gradle", version.ref = "agp" }
+dokkaGradlePlugin = { module = "org.jetbrains.dokka:dokka-gradle-plugin", version = "1.7.10" }
+grgitGradlePlugin = { module = "org.ajoberstar.grgit:grgit-gradle", version= "5.0.0" }
+kotlinGradlePlugin = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "kotlin" }
+mavenPublishGradlePlugin = { module = "com.vanniktech:gradle-maven-publish-plugin", version = "0.21.0" }
+spotlessGradlePlugin = { module = "com.diffplug.spotless:spotless-plugin-gradle", version = "6.8.0" }
+versionsGradlePlugin = { module = "com.github.ben-manes:gradle-versions-plugin", version = "0.42.0" }
+
+# Regular dependencies
+androidxAnnotations = { module = "androidx.annotation:annotation", version = "1.3.0" }
+androidxComposeCompiler = { module = "androidx.compose.compiler:compiler", version = "1.1.1" }
+androidxLifecycleCommon = { module = "androidx.lifecycle:lifecycle-common", version = "2.4.0" }
+assertj = { module = "org.assertj:assertj-core", version = "3.23.1" }
+bytebuddyAgent = { module = "net.bytebuddy:byte-buddy-agent", version.ref = "bytebuddy" }
+bytebuddyCore = { module = "net.bytebuddy:byte-buddy", version.ref = "bytebuddy" }
+guava =  { module = "com.google.guava:guava", version = "31.0.1-jre" }
+jcodecCore = { module = "org.jcodec:jcodec", version.ref = "jcodec" }
+jcodecJavase = { module = "org.jcodec:jcodec-javase", version.ref = "jcodec" }
+junit = { module = "junit:junit", version = "4.13.2" }
+kotlinBom = { module = "org.jetbrains.kotlin:kotlin-bom", version.ref = "kotlin" }
+kotlinxCoroutinesCore = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-core", version = "1.6.2" }
+kxml2 = { module = "kxml2:kxml2", version = "2.3.0" }
+moshiAdapters = { module = "com.squareup.moshi:moshi-adapters", version.ref = "moshi" }
+moshiCore = { module = "com.squareup.moshi:moshi", version.ref = "moshi" }
+moshiKotlinCodegen = { module = "com.squareup.moshi:moshi-kotlin-codegen", version.ref = "moshi" }
+testParameterInjector = { module = "com.google.testparameterinjector:test-parameter-injector", version = "1.8" }
+truth = { module = "com.google.truth:truth", version = "1.1.3" }
+toolsCommon = { module = "com.android.tools:common", version = "27.1.2" }
+toolsLayoutlib = { module = "com.android.tools.layoutlib:layoutlib-api", version = "27.2.2" }
+toolsNinepatch = { module = "com.android.tools:ninepatch", version = "30.1.2" }
+toolsSdkCommon = { module = "com.android.tools:sdk-common", version = "26.6.4" }
```

Comment:
tip: if you use dashes, you can keep the usages nicely nested

`tools-common = {` will be `api(tools.common)`

I think this is what John meant when he asked for `plugins` prefix.

`androidGradlePlugin` -> `plugins-android=` -> `api(plugins.android)`

Reply:
> tip: if you use dashes, you can keep the usages nicely nested and grouped

Thanks for the suggestion, but I don't see it as a beneficial thing.

> I think this is what John meant when https://github.com/cashapp/paparazzi/pull/522#discussion_r933698236 for plugins prefix.

I chatted offline, and what John wanted was not to use `plugins {}` API and instead use `buildscript {}` to load plugins.

Reply:
> I think this is what John meant when https://github.com/cashapp/paparazzi/pull/522#discussion_r933698236 for plugins prefix.

> Thanks for the suggestion, but I don't see it as a beneficial thing.

Yep, that's correct.  We'd prefer the dashes.